programs/vote/src/vote_state/mod.rs

```diff
@@ -877,8 +877,27 @@ pub fn update_commission<S: std::hash::BuildHasher>(
     vote_account: &mut BorrowedAccount,
     commission: u8,
     signers: &HashSet<Pubkey, S>,
+    epoch_schedule: &std::sync::Arc<EpochSchedule>,
+    clock: &std::sync::Arc<Clock>,
```

Comment:
nit: rather than take in the `Arc`, can you change these to the ref? ie
```suggestion
    epoch_schedule: &EpochSchedule,
    clock: &Clock,
```
I typically let the compiler tell me what to do to go from `Arc` to a normal ref, but I think you'll need to call into the function with `sysvar_cache.get_epoch_schedule()?.as_ref()`